packages/asset-swapper/src/utils/quote_requestor.ts

```diff
@@ -150,6 +156,21 @@ export class QuoteRequestor {
                 return false;
             }
 
+            if (order.takerAddress !== _opts.takerAddress) {
+                this._warningLogger(`Unexpected takerAddress in RFQ-T order, filtering out: ${JSON.stringify(order)}`);
+                return false;
+            }
+
+            const minExpiry = Math.round((Date.now() + this._expiryBufferMs) / constants.ONE_SECOND_MS);
```

Comment:
It seems weird to take in a `milliseconds` param just round it to `seconds` -- esp. since this would round down `400 ms` to `0 seconds`.  What do you think about just accepting the param in `seconds`?

Reply:
I cleaned this up a bunch here: https://github.com/0xProject/0x-monorepo/pull/2574/commits/125c53827bbde841a0d978c405452a14008dffd6

Let me know what you think.

Reply:
It still seems weird to take in `ms` and then round to `seconds`, esp when `400 ms` can round to `0` seconds.    That being said -- I see the expiry buffer is already defined in `ms` (expiryBufferMs), so perhaps we can't get around this.